Swift Code Challenge/Services/ApiRequestAsync.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ApiRequestAsync.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 30/06/22.
+//
+
+import Foundation
+
+class ApiRequestAsync {
+    
+    static func fetchUser() async throws -> [Person] {
```

Comment:
A great place for documentation 
```suggestion
    /// Fetches users from the API
    static func fetchUsers() async throws -> [Person] {
```